src/modules/governance/api.ts

```diff
@@ -1,11 +1,11 @@
 import BigNumber from 'bignumber.js';
```

Comment:
Would be good to keep logic pointing the graph. Keep the version of this file as one from development.